WordleGame/WordleGame/Views/Letter/GameFieldView.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  GameFieldView.swift
+//  WordleGame
+//
+//  Created by Alex on 3.05.22.
+//
+
+import Foundation
+import UIKit
+
+class GameFieldView: UIStackView {
```

Comment:
The field looks like this on iPhone 13. Is it supposed to look like that?

![Simulator Screen Shot - iPhone 13 - 2022-05-12 at 00 52 54](https://user-images.githubusercontent.com/25828050/167954512-3ba0bd84-7486-4008-aa23-f255112761ef.png)

Reply:
Yes, initially it's normal =) I will edit design of app later.